packaging/install/030-ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: strimzi-access-operator
+  labels:
+    app: strimzi-access-operator
+subjects:
+  - kind: ServiceAccount
+    name: strimzi-access-operator
+    namespace: strimzi
```

Comment:
Should we maybe discuss the default namespace? I wonder if it should be in sync with the other repos (`myproject`).

Reply:
I chose this namespace knowing that it wouldn't match the other repos, since `myproject` is an unusual name for a namespace. But if it's required to match the other repos I can change it. Another option would be to leave `strimzi` here but put `myproject` as the namespace of the Kafka cluster in the example files, since you will be able to run this operator in a different namespace to the Strimzi cluster operator if you wanted.

Reply:
I don't know if it is required. But it might make sense to use the same defaults everywhere. I don't have a strong opinion about it.

Reply:
One more thought on the namespace ... if you want to default to watching whole cluster, you can approach it as the Drain Cleaner does it:
* Have the Namespace resource in the installation files
* Use `strimzi-access-operator`

If you watch whole cluster, the namespace is less important the in operators. So if we bundle it including the namespace resource, it abstracts also the need to do the `sed` stuff to change namespace etc. We can give the user the install files as they are and if they want to use anohter namespace, they can do whatever they want with the installation files.

Reply:
I like this suggestion, so have changed to use this approach